ur_robot_driver/doc/features.md

```diff
@@ -1,35 +1,31 @@
-| continue trajectories after protective stop           | -                | yes                   |
-| panel interaction in between possible                 | no<sup>1</sup>   | yes                   |
-| get and set IO states                                 | yes              | yes               |
-| use tool communication on e-series                    | -                | yes                   |
-| use the driver without a teach pendant necessary      | -                | yes               |
-| support of CB2 robots                                 | yes              | -                     |
-| trajectory extrapolation on robot on missing packages | no<sup>2</sup>   | yes                   |
-| use ROS as drop-in for TP-programs                    | -                | yes   |
-| headless mode                                         | yes              | yes   |
-| extract calibration from robot                        | -                | yes                   |
-| send custom script commands to robot                  | yes              | yes               |
-| ROS 2 support                                         | ?                | (planned)<sup>3</sup> |
-| Reconnect on a disconnected robot                     | yes              | yes           |
+| Feature                                               | this_driver           |
+| ---                                                   | ---                   |
+| joint position-based control                          | yes                   |
+| scaled joint position-based control                   | yes                   |
+| joint-velocity-based control                          | yes                   |
+| Cartesian position-based control                      | yes                   |
+| Cartesian twist-based control                         | yes                   |
+| Trajectory forwarding for execution on robot          | yes                   |
+| reporting of tcp wrench                               | yes                   |
+| reporting of tcp wrench in tcp link                   | yes                   |
+| pausing of programs                                   | yes                   |
+| continue trajectories after EM-Stop resume            | yes                   |
+| continue trajectories after protective stop           | yes                   |
+| panel interaction in between possible                 | yes                   |
+| get and set IO states                                 | yes               |
+| use tool communication on e-series                    | yes                   |
+| use the driver without a teach pendant necessary      | yes               |
+| support of CB2 robots                                 | -                     |
+| trajectory extrapolation on robot on missing packages | yes                   |
+| use ROS as drop-in for TP-programs                    | yes   |
```

Comment:
Suggest a adding a link for this as well

Reply:
See fb6c3ee